packages/react-sdk/src/components/separator.ws.tsx

```diff
@@ -0,0 +1,31 @@
+import { DashIcon } from "@webstudio-is/icons";
+import type { WsComponentMeta, WsComponentPropsMeta } from "./component-type";
+import props from "./__generated__/blockquote.props.json";
+
+const presetStyle = {
+  height: {
+    type: "keyword",
+    value: "1px",
+  },
+  backgroundColor: {
+    type: "keyword",
+    value: "gray",
+  },
+  border: {
+    type: "keyword",
+    value: "none",
+  },
+} as const;
+
+export const meta: WsComponentMeta = {
+  type: "embed",
```

Comment:
this is a little weird for a separator, I would have called it "void" but image, input etc are also void https://developer.mozilla.org/en-US/docs/Glossary/Void_element

Reply:
we condidered that name before, but it was weird to call image or input void from general perspective, but maybe we should